x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/action/GetQueryRulesetAction.java

```diff
@@ -79,11 +84,37 @@ public boolean equals(Object o) {
         public int hashCode() {
             return Objects.hash(rulesetId);
         }
+
+        @Override
+        public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+            builder.startObject();
+            builder.field(RULES_ID.getPreferredName(), rulesetId);
+            builder.endObject();
+            return builder;
+        }
+
+        private static final ConstructingObjectParser<Request, String> PARSER = new ConstructingObjectParser<>(
+            "get_query_ruleset_request",
+            false,
+            (p) -> {
+                return new Request((String) p[0]);
+            }
+
+        );
+        static {
+            PARSER.declareString(constructorArg(), RULES_ID);
+        }
+
+        public static Request parse(XContentParser parser, String name) {
+            return PARSER.apply(parser, name);
+        }
+
     }
 
     public static class Response extends ActionResponse implements ToXContentObject {
 
         private final QueryRuleset queryRuleset;
+        private static final ParseField QUERY_RULE_SET = new ParseField("queryRuleset");
```

Comment:
Can we name this `QUERY_RULESET_FIELD` to be consistent with how we define fields elsewhere?  I suppose since this field isn't actually used it's OK to keep the value`queryRuleset` as we do that for `searchApp`.